content/donate.adoc

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: "Donate to Jenkins"
+---
+
+NOTE: You can donate directly through the
+link:https://co.clickandpledge.com/advanced/default.aspx?wid=46160[SPI donation
+page] (donation in US dollars) or via the
+link:http://www.ffis.de/Verein/donations.html[ffis.de donation page] (donation
+in Euros) if you do not need the context below
+
+
+
+The Jenkins project is affiliated with link:http://www.spi-inc.org[Software in
+the Public Interest], a non-profit organization. Via SPI we can accept
+link:https://co.clickandpledge.com/advanced/default.aspx?wid=46160[donations
+online] in US dollars. For other ways to donate, refer to the
+link:http://spi-inc.org/donations[donations page on the SPI website].
+
+
+For European residents, we can also accept donations through
+link:http://www.ffis.de/Verein/donations.html[ffis.de] via bank transfer in
+Euro.
+As ffis.de is a charitable association, this donation is tax deducitble under
+in Germany.
+
+NOTE: If you donate via ffis.de, please write to `donation[at]ffis.de` to
+ensure that your donation will be earmarked for the Jenkins project.
+
+
+== Why donate?
+
+Your contributions help us keep Jenkins going by paying project expenses, such
+as:
+
+
+* Operating expenses such as:
+** SSL certificates on our websites
+** Covering network transit costs (not including that incurred by mirrors)
+** Equipment/replacement hardware
+* A portion of contributions help fund SPI, Inc, which acts as the legal
+  umbrella organization for holding copyrights, etc.
+
+
+Your contribution is *not* used for paying personnel; all the contributors work
+on a voluntary basis.
+
+
+To show our appreciation, a manually maintained list of
+link:https://wiki.jenkins-ci.org/display/JENKINS/Donors[past donors] is
+available.
+
+== Friend of Jenkins
+
+In showing our appreciation, we'll send out a special
```

Comment:
We need to confirm with @kohsuke that this actually still exists. The past donors page isn't kept up to date.